code/modules/clothing/head/helmet.dm

```diff
@@ -80,6 +77,12 @@
 	can_toggle = 1
 	toggle_cooldown = 20
 	activation_sound = 'sound/items/WEEOO1.ogg'
+	
+/obj/item/clothing/head/helmet/justice/attack_self()

```

Comment:
activation_sound implies a one time thing, not an "active_sound." If any helmets were to ever get a sound (and only helmet subtypes, which aren't many) it'd make sense for them to play the sound just once, not try to play it constantly.

No other helmet but the justice one attempts to play a constant sound.